### PR TITLE
Consolidate NVP requests into single request

### PR DESCRIPTION
**Pull request by marcaaron**
cc @stitesExpensify 

### Fixed Issues
$ https://github.com/Expensify/App/issues/4883

### Tests

**Note:** Going to keep the testing minimal here as I'm assuming if a few of these NVPs work then the rest of them should work as well. I didn't bother to set the private NVPs in this case, but I think there should be no issue with them at all since they will be handled much like the non private NVPs.

1. Added the following public NVPs via `expensify.com.dev` JS console so that they would appear when fetched
```js
NVP.set('expensify_migration_2020_04_28_RunKycVerifications', '2020-01-13')
NVP.set('expensify_ACHData_throttled', '2021-09-09 18:17:58');
```
2. Navigated to `https://localhost:8080/bank-account` with the network tab open to capture http response data
3. Located the `Get` command
4. Verified the NVPs appear
5. Verified the logic in the API handler works since we are handling the error about throttled account

### QA Steps

1. Navigate to `/bank-account` and verify the initial screen loads without any issue

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [ ] Android

### CODE REVIEW

**Review by stitesExpensify — DISMISSED** [review]

**Review by Julesssss — COMMENTED** [review]
Nice. Will review on Mon. morning if it's ready and hasn't merged by then.

**Comment by stitesExpensify** [comment]
Linting failed on this :(

**Comment by marcaaron** [comment]
I'll pick this back up in a bit. Presently trying to get to the bottom of this -> https://github.com/Expensify/App/issues/4959 ⚰️

**Comment by marcaaron** [comment]
Still working out test steps for this. I _think_ maybe we can just set all these NVPs manually and then verify they are coming back and being handled properly. Then just have QA verify they can still create accounts vs. have them test these weird error states.

**Comment by stitesExpensify** [comment]
> Still working out test steps for this. I think maybe we can just set all these NVPs manually and then verify they are coming back and being handled properly. Then just have QA verify they can still create accounts vs. have them test these weird error states.

That sounds like a good strat to me

**Comment by marcaaron** [comment]
This one is ready now. Thanks !

File: src/libs/actions/BankAccounts.js [hunk 1]
```diff
@@ -363,54 +362,28 @@ function fetchFreePlanVerifiedBankAccount(stepToOpen) {
...
-            ])
-                .then(([
-                    failedValidationAttemptsResponse,
-                    kycVerificationsMigrationResponse,
-                    achDataThrottledResponse,
-                    bankAccountListResponse,
-                    throttledBankAccountGetResponse,
-                ]) => {
+            API.Get({
+                returnValueList: 'nameValuePairs,bankAccountList',
```
Comment (stitesExpensify) [inline src/libs/actions/BankAccounts.js:366]: Can you add a space here? this just looks ugly 😅

Comment (marcaaron) [reply src/libs/actions/BankAccounts.js:366]: For you? Yea, I can do that.

Comment (stitesExpensify) [reply src/libs/actions/BankAccounts.js]: 😘

**Review by stitesExpensify — APPROVED** [review]

**Pull request merged**